src/04-modality-specific-files/05-task-events.md

```diff
@@ -129,16 +197,17 @@ of example.
    "identifier": {
        "LongName": "Unique identifier from Karolinska (KDEF) database",
        "Description": "ID from KDEF database used to identify the displayed image"
-   }
+   },
+    "StimulusPresentation": {
+        "OperatingSystem": "Linux-Ubuntu 18.04.5 LTS",
+        "SoftwareName": "Psychtoolbox",
+        "SoftwareRRID": "SCR_002881",
+        "SoftwareVersion": "3.0.14",
+        "Code": "doi:10.5281/zenodo.3581316"
+    }
 }
-```	   
+```
 
-For multi-echo files, the `*_events.tsv` file is applicable to all echos of particular
-run:
+<!-- Link Definitions -->
```

Comment:
Just noticed that some other pages have this "heading" for the link definition section:

```
[]: <> (################)
[]: <> (Link definitions)
[]: <> (################)
```

Do we want to use it here as well?

Reply:
these sections were recently removed (https://github.com/bids-standard/bids-specification/pull/587)

and replaced with the `<!-- Link definitions -->` that we use in this PR, so no changes needed :)